pkgs/applications/accessibility/mousetweaks/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitLab
+, autoreconfHook, pkgconfig, makeWrapper
+, glib, gtk3, gsettings-desktop-schemas
+, libX11, libXtst, libXfixes, libXcursor
+}:
+stdenv.mkDerivation rec {
+  pname = "mousetweaks";
+  version = "3.32.0";
+
+  src = fetchFromGitLab {
```

Comment:
Please switch to fetching from gnome mirror, it is geographically distributed:

https://github.com/NixOS/nixpkgs/blob/5654a3d968b4bcb2a718537711eb2ed2cb418a8f/pkgs/applications/misc/orca/default.nix#L16-L19

It should also allow us to get rid of `autoreconfHook`.